libraries/DateTime/contracts/DateTime.aes

```diff
@@ -39,88 +32,27 @@ contract DateTime =
       minute_in_seconds = 60
       }
 
+  private function require(expression : bool, error : string) =
```

Comment:
Maybe it will be good if all private functions will be grouped together with all private functions. Same with public functions.